submissions/andrewklmn/memory-pair-game/app.css

```diff
@@ -0,0 +1,123 @@
+body {
+  width: 90%;
+  max-width: 450px;
+
+  background-color: lightgray;
+  
+  font-family: 'Courier New', Courier, monospace;
+  margin: auto;
+}
+
+.info{
+  display: flex;
+  justify-content: center;
+  padding-top: 15px;
+  padding-bottom: 10px;
+
+  width: 100%;
+
+  font-size: 1.2em;
+  text-align: center;
+}
+
+.gameboard{
+  display: grid;
+  grid-template-columns: 25% 25% 25% 25%;
+  grid-template-rows: 25% 25% 25% 25%;
+
+  height: 90vw;
+  max-height: 450px;
+
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width:100%;
+  cursor: pointer;
+}
+
+.guessed {
+  opacity: 0.3;
+}
+
+.hide {
+  opacity: 0;
```

Comment:
If we hide the control wrapper, we can find it with the cursor and reset the game again.
Use `display: none` and paste this rule after `.control`